modules/unsupported/geojson/src/main/java/org/geotools/geojson/DelegatingHandler.java

```diff
@@ -53,50 +53,61 @@
     protected static NullHandler NULL = new NullHandler();
     protected static NullHandler UNINITIALIZED = new NullHandler();
 
-    protected static List NULL_LIST = Collections.unmodifiableList(new ArrayList(0));
+    protected static List<String> NULL_LIST =
```

Comment:
This could have been corrected as Collections.emptyList(), it's a generics safe, empty, unmodifiable list. But it does not hurt to leave it this way. Merging ;-)